packages/@react-spectrum/list/stories/ListView.stories.tsx

```diff
@@ -160,70 +182,64 @@ storiesOf('ListView', module)
     </ListView>
     )
   )
-  .add('falsy ids as keys', () => (
-    <FalsyIds />
+  .add('falsy ids as keys', args => (
+    <FalsyIds {...args} />
   ))
-  .add('empty list', () => (
-    <ListView width="300px" height="300px" renderEmptyState={renderEmptyState}>
+  .add('empty list', args => (
+    <ListView width="300px" height="300px" renderEmptyState={renderEmptyState} {...args}>
       {[]}
     </ListView>
   ))
-  .add('loading', () => (
-    <ListView width="300px" height="300px" loadingState="loading">
+  .add('loading', args => (
+    <ListView width="300px" height="300px" loadingState="loading" {...args}>
       {[]}
     </ListView>
   ))
-  .add('loadingMore', () => (
-    <ListView width="300px" height="300px" loadingState="loadingMore">
+  .add('loadingMore', args => (
+    <ListView width="300px" height="300px" loadingState="loadingMore" {...args}>
       <Item textValue="Adobe Photoshop">Adobe Photoshop</Item>
       <Item textValue="Adobe Illustrator">Adobe Illustrator</Item>
       <Item textValue="Adobe XD">Adobe XD</Item>
     </ListView>
   ))
-  .add('async listview loading', () => (
-    <AsyncList />
+  .add('async listview loading', args => (
+    <AsyncList {...args} />
   ))
-  .add('async listview loading with actions', () => (
-    <AsyncList withActions />
+  .add('async listview loading with actions', args => (
+    <AsyncList withActions {...args} />
   ))
-  .add('density: compact', () => (
-    <ListView width="250px" density="compact">
-      <Item textValue="Adobe Photoshop">Adobe Photoshop</Item>
-      <Item textValue="Adobe Illustrator">Adobe Illustrator</Item>
-      <Item textValue="Adobe XD">Adobe XD</Item>
+  .add('dynamic items + renderEmptyState', args => (<EmptyTest {...args} />))
+  .add('with ActionBar', args => <ActionBarExample {...args} />)
```

Comment:
The controls don't work with this story because `ActionBarExample` wasn't updated to spread them. Is that something to fix now or with the CSF 3.0 refactor?